bridge/test/observers/burn-event-observer.spec.ts

```diff
@@ -254,6 +267,223 @@ describe(EthereumBurnEventObserver.name, () => {
             ]);
         });
 
+        it("should post slack message every events of Multi-Planet Request - Odin", async () => {
+            const ncgRecipient = "0x6d29f9923C86294363e59BAaA46FcBc37Ee5aE2e";
+            function makeEvent(
+                ncgRecipient: string,
+                amount: number,
+                txId: TxId
+            ): EventData & TransactionLocation {
+                return {
+                    blockHash: "BLOCK-HASH",
+                    address: "0x4029bC50b4747A037d38CF2197bCD335e22Ca301",
+                    logIndex: 0,
+                    blockNumber: 0,
+                    event: "Burn",
+                    raw: {
+                        data: "",
+                        topics: [],
+                    },
+                    signature: "",
+                    transactionIndex: 0,
+                    transactionHash: txId,
+                    txId: txId,
+                    returnValues: {
+                        _sender: "0x2734048eC2892d111b4fbAB224400847544FC872",
+                        // Multi-Planet Request Type
+                        _to: (planetIds.odin + ncgRecipient.slice(2)).padEnd(
+                            66,
+                            "0"
+                        ),
+                        amount: amount,
+                    },
+                };
+            }
```

Comment:
It seems like a copied function from `"should post slack message every events" test case. And only the `_to`  seems different. How about extracting the function with a new parameter `to` and sharing the logic?